iphone/package.json

```
@@ -19,7 +19,7 @@
 	"minIosVersion": "10.0",
 	"minWatchosVersion": "2.0",
 	"vendorDependencies": {
-		"xcode": ">=9.0 <=12.x",
+		"xcode": ">=11.0 <=12.x",
 		"ios sdk": ">=10.0 <=14.x"
```

1. With Xcode 11, iOAS SDK is 13. So update-
```suggestion
		"ios sdk": ">=13.0 <=14.x"
```
2. As we are targeting minimus iOS SDK 13.0, we need to remove all macros IS_SDK_IOS_12 and IS_SDK_IOS_13. Because these will be irrelevant.